scilpy/tractanalysis/todi.py

```diff
@@ -90,15 +142,36 @@ def mask_todi(self, mask):
         self.mask = new_mask
         self.todi = new_todi
 
-    def smooth_todi_dir(self, order=1):
-        assert(order >= 1)
+    def smooth_todi_dir(self, order=2):
+        """Smooth orientations on the sphere.
+
+        Smooth the orientations / distribution on the sphere.
+        Important for priors construction of BST.
+
+        Parameters
+        ----------
+        order : int, optional
+            Exponent blurring factor, based on the dot product
+            (default 2).
```

Comment:
Why 2 now ?

Reply:
it's the default used in the BST paper, its also the one used in TODI